app/main.py

```diff
@@ -1 +1,67 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+
+    def __init__(self,
+                 name: str) -> None:
+        super().__init__(name)
+
+        for i in ["JavaScript", "HTML", "CSS"]:
+            self.learn_skill(i)
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+
+    def __init__(self,
+                 name: str) -> None:
+        super().__init__(name)
+
+        for i in ["Python", "SQL", "Django"]:
+            self.learn_skill(i)
+
+    def create_powerful_api(self) -> str:
+        print(f"{self.name} is creating an API...")
+        return "http://127.0.0.1:8000"
+
+
+class AndroidDeveloper(SoftwareEngineer):
+
+    def __init__(self,
+                 name: str) -> None:
+        super().__init__(name)
+
+        for i in ["Java", "Android studio"]:
+            self.learn_skill(i)
+
+    def create_smooth_mobile_app(self) -> str:
+        print(f"{self.name} is creating a mobile app...")
+        return "Ads every three swipes"
+
+
+class FullStackDeveloper(BackendDeveloper, FrontendDeveloper):
+
+    def __init__(self,
+                 name: str) -> None:
+        super().__init__(name)
+        print(self.skills)
+
+    def create_web_application(self) -> None:
+        print(f"{self.name} started creating a web application...")
+        self.create_powerful_api()
+        self.create_awesome_web_page()
+
+
+full_stack_dev = FullStackDeveloper("Tom")
+sa = AndroidDeveloper("Sam")
```

Comment:
This should be removed as it's not a part of module.

Reply:
Ok